src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/PhysicalAddress.cs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+
+namespace System.Net.NetworkInformation
+{
+    public class PhysicalAddress
+    {
+        private byte[] _address = null;
+        private bool _changed = true;
```

Comment:
Does "_changed" really just mean "_hashNotComputed"?  I don't see anywhere else setting it back to true.